playerSDK/src/main/java/com/kaltura/playersdk/PlayerViewController.java

```diff
@@ -87,6 +70,7 @@
     private HashMap<String, ArrayList<KPlayerEventListener>> mKplayerEventsMap = new HashMap<String, ArrayList<KPlayerEventListener>>();
     private HashMap<String, KPlayerEventListener> mKplayerEvaluatedMap = new HashMap<String, KPlayerEventListener>();
     private KPlayerJsCallbackReadyListener mJsReadyListener;
+    private JSONObject nativeActionParams;
```

Comment:
For now we are using it for the native share, but I will refactor this approach similar to the iOS